File: 01/01.exs
gnomes =
  File.read!(Path.expand("~/co/aoc-2022/01/input.txt"))
  |> String.split("\n\n")
  |> Enum.map(fn x -> String.split(x, "\n", trim: true) |> Enum.map(&String.to_integer/1) end)
  |> Enum.map(&Enum.sum/1)
  |> Enum.sort
  |> Enum.reverse

# 70698
IO.inspect(Enum.max(gnomes))

# 206643
IO.inspect(Enum.sum(Enum.take(gnomes, 3)))

File: 01/01.py
f = open("input.txt")

gnomes = []
curr = []
for l in f.readlines():
    if l.strip() != '':
        curr.append(int(l.strip()))
    else:
        gnomes.append(curr)
        curr = []
totals = map(lambda x: sum(x), gnomes)
totals.sort()
totals = totals[::-1]

# 70698
print("Part 1: ", totals[0])

# 206643
print("Part 2: ", sum(totals[0:3]))
